android/quest/src/androidTest/java/org/smartregister/fhircore/quest/integration/ui/shared/components/ActionableButtonTest.kt

```diff
@@ -100,6 +100,17 @@ class ActionableButtonTest {
       .performClick()
   }
 
+  @Test
+  fun testActionableButtonRendersAndCopyText() {
+    setCopyButtonContent("false")
+
+    composeRule
+      .onNodeWithText("Copy Button Text", useUnmergedTree = true)
+      .assertExists()
+      .assertIsDisplayed()
+      .performClick()
```

Comment:
Should there be a verification done for this perform click action?

Reply:
I will update the test to test on click function instead